new-frontiers-14/frontier-station-14
# Add a bucket and mop slot to the Janicart

## About the PR
-Adds a bucket and mop slot to the Janicart, upgrading the arsenal of our valiant janitorial warriors to better combat the messy nature of the vastness of space.

## Why / Balance
No bigtime balance issues. Came out of some idle chatter on Discord.

https://discord.com/channels/1123826877245694004/1123851369913266256/1314977814918860810
https://discord.com/channels/1123826877245694004/1127687656084611214/1315002359511781476

## How to test
Spawn a Janicart NF variant in, check functions. Check out mapped locations of the cart to see the NF version is there. Check out dungeon spawners and Janicart crates and see they spawn the NF version.

## Media
![crate](https://github.com/user-attachments/assets/b39e2648-8e8a-4f16-ad54-0d856f379288)
![examine](https://github.com/user-attachments/assets/9b702f3a-b365-4675-a8b1-1fd09f17f889)
![mopeject](https://github.com/user-attachments/assets/2966284d-04ec-4d78-b283-c011041cbdde)
![mopinsert](https://github.com/user-attachments/assets/0fc87c71-9b2a-476c-9b3c-f6946926f797)
![moptry](https://github.com/user-attachments/assets/17ab03c2-cb07-4fed-b54d-46d153fb7b98)


## Requirements



## Breaking changes
Nothing really

**Changelog**
:cl:
- tweak: Add slots to the Janicart vehicle to give it feature parity with the janitor trolley and make things more convenient for practicioners of the janitorial arts
- add: Research target that unlocks crafting of Janicart flatpacks
- remove: Janicart crates no longer purchaseable at cargo
